packages/nextjs/services/database/grants.ts

```diff
@@ -166,9 +170,21 @@ export const reviewGrant = async ({ grantId, action, txHash, txChainId, note }:
   }
 };
 
-export const updateGrant = async (grantId: string, grantData: Partial<GrantData>) => {
+export const updateGrant = async (grantId: string, grantData: Partial<GrantData>, private_note?: string) => {
   try {
     await getGrantsDoc(grantId).update(grantData);
+
+    if (private_note === undefined) return;
```

Comment:
Could you explain this @technophile-04? When is this going to happen?

I think we are always getting an empty string here, right? (when the note is empty)

Merging this for now! but maybe we can discuss in the issue/another PR/telegram :D

Reply:
well, or discuss here too! haha

Reply:
Ohh my bad :(, yup this should be removed 

Actually here,
https://github.com/BuidlGuidl/grants.buidlguidl.com/blob/a8e1a7b84ae99525f0b4066b089542286f278f57/packages/nextjs/app/api/grants/%5BgrantId%5D/route.ts#L41

I had this : 

```ts
await updateGrant(
  grantId,
  { title, description, askAmount },
  private_note?.trim().length ? private_note : undefined,
);
```

With `if (private_note === undefined) return;` and above code snippet, I was just trying to save one write when : 

Admin see's fresh new grant and want's to just edit title / descp / askAmount but don't want to add note (Initally note value is empty and this would have saved us one write to subcollection) 

But then realised, what if admin had already added some note and now he wants to remove the note, he would simply remove note from EditModal and send empty string but with above code snippet and this `if` condition it will never delete / empty the note and keep the note as it. 

So I removed above code snippet but forgot remove this `if (private_note === undefined) return;` from here :(

Reply:
Just added to my notes and will remove this in next PR I make while tackling some other issue 🙌

Reply:
That makes sense!! Thank you Shiv